spec/api-deprecations-spec.js

```diff
@@ -58,19 +62,19 @@ describe('deprecations', () => {
 
     let value = 0
     let o = { [newPropertyName]: value }
-    assert.strictEqual(typeof o[oldPropertyName], 'undefined')
-    assert.strictEqual(typeof o[newPropertyName], 'number')
+    expect(o[oldPropertyName]).to.be.undefined()
+    expect(o[newPropertyName]).to.be.a('number')
```

Comment:
I guess we expect the `oldPropertyName` to not exist at all rather than exist and have value of `undefined`, right?

```js
expect(o).to.not.have.a.property(oldPropertyName)
expect(o).to.have.a.property(newPropertyName).that.is.equal(value)
```